.drone.yml

```diff
@@ -3643,6 +3643,7 @@ steps:
   - apk add --no-cache make
   - chown -R $UID:$GID /go
   - cd /go/src/github.com/gravitational/teleport
+  - echo "${WINDOWS_SIGNING_CERT}" | base64 -d > windows-signing-cert.pfx
   - make -C build.assets release-amd64
```

Comment:
I would also add a line here to explicitly `rm -f windows-signing-cert.pfx` after the build is complete. This is good hygiene in general (deliberately removing key material as soon as it's been used) and will reduce the risk of any change to a subsequent stage accidentally uploading the cert.

We do actually use `/tmpfs` as an in-memory `EmptyDir` elsewhere in the pipeline to try and guarantee the ephemerality of key material and prevent things like this happening - see https://github.com/gravitational/teleport/blob/master/.drone.yml#L4329-L4331

I'm unsure whether this is actually improving security at all, though, so I'm not going to stipulate that it's required here.

Reply:
That's a good thought, I've tweaked it to write the cert to a tmpfs and then symlink it into the build directory.

Reply:
Walking that back, something about my implementation was not playing nice in Drone. I've opted not to waste more time debugging that and am now just removing the file following a successful build as you recommended.